### ISSUE TITLE
NVDA doesn't narrate the question when the dropdown is selected

### ISSUE THREAD

**Issue by lemmyadams** [adaptlearning/adapt-contrib-matching#206]
### Subject of the issue/enhancement/features
NVDA doesn't narrate the question when the dropdown is selected

### Steps to reproduce
Use NVDA
Navigate to the dropdown

### Expected behaviour
It should narrate the question, as well as saying 'Please select and option' or similar, so readers are aware of the question.

### Actual behaviour
It doesn't read out the question

On other screen readers, such as the built-in Mac one, it reads out the question
